src/main/target/CC3D/config.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <platform.h>
+
+#include "config/config_master.h"
+#include "config/feature.h"
+#include "sensors/battery.h"
+
+void targetValidateConfiguration(master_t *config)
+{
+	UNUSED(config);
+
+#if defined(DISPLAY) && defined(USE_UART3)
```

Comment:
I think DISPLAY is gone for good from CC3D (now even I2C which is needed by DISPLAY is gone), so this can probably be removed.